src/connector/src/sink/formatter/mod.rs

```diff
@@ -225,6 +249,22 @@ macro_rules! dispatch_sink_formatter_impl {
             SinkFormatterImpl::AppendOnlyJson($name) => $body,
             SinkFormatterImpl::AppendOnlyProto($name) => $body,
             SinkFormatterImpl::UpsertJson($name) => $body,
+            SinkFormatterImpl::UpsertAvro($name) => $body,
+            SinkFormatterImpl::DebeziumJson($name) => $body,
+            SinkFormatterImpl::AppendOnlyTemplate($name) => $body,
+            SinkFormatterImpl::UpsertTemplate($name) => $body,
```

Comment:
what template mean here?

Reply:
`format upsert encode template (key_format = 'V1 is: {v1}', value_format = 'V2 is: {v2}')` initially introduced by redis sink.